### PR TITLE
Adding CVE-2014-3626

### PR DESCRIPTION
**Pull request by DellEMCProductSecurity**

### CODE REVIEW

File: 2014/3xxx/CVE-2014-3626.json [hunk 1]
```diff
@@ -1,8 +1,32 @@
 {
```
Comment (cve-team) [inline 2014/3xxx/CVE-2014-3626.json]: JSON: Expecting property name: line 65 column 3 (char 1719)

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

File: 2014/3xxx/CVE-2014-3626.json [hunk 2]
```diff
@@ -11,8 +35,32 @@
...
+      "reference_data" : [
+         {
+            "url" : "http://grails.org/plugin/resources"
+         }
+      ]
+   },
+   "credits" : {	
+			"role" : {
+      				"discoverer" : "Mr. Preecha Jeangsiriphan"     								      							     						
+      			},
```
Comment (cve-team) [inline 2014/3xxx/CVE-2014-3626.json]: Remove the comma at the end of line 64.

George

File: 2014/3xxx/CVE-2014-3626.json [hunk 3]
```diff
@@ -11,8 +35,32 @@
...
+               }
+            ]
          }
       ]
-   }
+   },
+   "references" : {
+      "reference_data" : [
+         {
+            "url" : "http://grails.org/plugin/resources"
```
Comment (cve-team) [inline 2014/3xxx/CVE-2014-3626.json]: By the way, it's not clear to me how this reference providence provenance. https://pivotal.io/security/cve-2014-3626 seems like a better reference to use.

George

Comment (DellEMCProductSecurity) [reply 2014/3xxx/CVE-2014-3626.json]: I will confirm and update accordingly. Apart from the comma and the reference, everything else is good, right?

Comment (cve-team) [reply 2014/3xxx/CVE-2014-3626.json]: The only other point to keep in mind is that the description itself should contain the product and version information generally as we use it to populate the associated CVE entry. In this case, the description is missing version info.

Other than that and the comma and reference, it looks good.

George

**Comment by cve-team** [comment]
Are you planning to update the description too?

**Comment by DellEMCProductSecurity** [comment]
No, the link provided gives the details of how the flaw is fixed, the description that I put in the file describes what the vulnerability is about.

**Comment by cve-team** [comment]
Per the CNA Rules, the description should include it (ie, "The [DESCRIPTION] field is a plain language field that should describe the vulnerability with sufficient detail as to demonstrate that the vulnerability is unique. The required information listed above should be included in the [DESCRIPTION], as well as other details the author feels are relevant or necessary to show uniqueness."

**Pull request closed without merging**